bench-tps/src/perf_utils.rs

```diff
@@ -0,0 +1,90 @@
+use {
+    crate::bench_tps_client::BenchTpsClient,
+    log::*,
```

Comment:
I think bench-tps is the only thing using sample_txs, so either modify it directly or remove the old one?

Reply:
Done on the part 1 PR: https://github.com/solana-labs/solana/pull/24208/commits/c0c6b2b3bdc9057b8149d4074ead1aa56eadeb0b